modules/setting/setting.go

```diff
@@ -226,12 +226,3 @@ func LoadSettingsForInstall() {
 	loadServiceFrom(CfgProvider)
 	loadMailerFrom(CfgProvider)
 }
-
-var uniquePaths = make(map[string]string)
-
-func fatalDuplicatedPath(name, p string) {
-	if targetName, ok := uniquePaths[p]; ok && targetName != name {
-		log.Fatal("storage path %q is being used by %q and %q and all storage paths must be unique to prevent data loss.", p, targetName, name)
```

Comment:
Then it comes to the same question:

* How many users really care about the startup logs .... I have answered many questions about "why something doesn't work" but the startup log already told users.
* What an user could do if they see such message. Any document or guideline?

Reply:
Let me throw the ball right back at you:
Yes, it is in no way a complete solution, I've understood that already.

However, what is better?
- Keeping a fix that while not perfect does work at scaring (some) people from shooting themselves in the foot
- Disabling this check completely because we don't have any idea what recovery measures are available

That's exactly the problem: Everyone who has this problem has to decide how to proceed best themselves.
Gitea does not know which file belongs where. That's the original problem we want to fix.
I don't see any other way than letting each affected admin decide what should be moved where.

I'm in favor of at least keeping the `log.Error`.
Originally, before you notified me that all Docker users have this problem by default, I would have even argued for `Fatal`, but that's a valid problem that we have caused ourselves.
As such, we cannot `Fatal` anymore, but we can at least make every user (that reads the docs) aware of the problem.

And in the meantime, we can work on a better fix.